### PR TITLE
Adds /version switch

### LINKED ISSUE #947: GitVersion CommandLine doesn't show its own version number
When running `GitVersion /?`, the output text doesn't show what the version is of GitVersion itself.

Running `GitVersion /?` and `GitVersion /l` should display the GitVersion version somewhere.

### PR DESCRIPTION
**Pull request by kjjuno**
Closes #947 

The version of GitVersion is now displayed if the /version switch is provided. The version is also included in the help page.

### CODE REVIEW

**Comment by asbjornu** [comment]
@kjjuno: Awesome! Do you think it would be possible to add a test or two for this as well?

**Comment by kjjuno** [comment]
For sure. I'll add some when I get a chance

**Comment by kjjuno** [comment]
For some reason the travis-ci build seems to be hung. Is there a way to kill it and start over. Did I do something in my unit tests that isn't supported on travis?

**Comment by asbjornu** [comment]
@kjjuno: I dunno what happened. I restarted the build now.

**Comment by kjjuno** [comment]
I'm guessing the dynamic assembly compile stuff doesn't work on linux. I may have to think of a different test.

**Comment by kjjuno** [comment]
I resolved the build issue by preventing this test from running on mono. The test compiles an assembly at runtime with a specific version, and then tests that the version is correct. The dynamic compilation code is not supported on mono.

**Comment by asbjornu** [comment]
@kjjuno: Perhaps you can use Mono.Cecil to compile the assembly instead? That should work on Mono just fine.

**Comment by kjjuno** [comment]
I've never used that before. I'll learn how to do that and update the pr

File: src/GitVersionExe.Tests/VersionWriterTests.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+            Assert.IsNotNull(asm);
+            Assert.AreEqual("1.0.0-beta0004", version);
+        }
+
+        private Assembly GenerateAssembly(Version fileVersion, string prereleaseInfo)
+        {
+            var definition = new AssemblyNameDefinition("test-asm", fileVersion);
+
+            var asmDef = AssemblyDefinition.CreateAssembly(definition, "test-asm", ModuleKind.Dll);
```
Comment (JakeGinnivan) [inline src/GitVersionExe.Tests/VersionWriterTests.cs]: Nice work figuring out how to do this, nice solution

**Comment by ghuntley** [comment]
Welcome back Jake!

**Comment by JakeGinnivan** [comment]
Thanks @ghuntley, first weekend in a month without a wedding or a bucks party on :)

File: src/GitVersionExe.Tests/VersionWriterTests.cs (hunk 1 above)
Comment (kjjuno) [inline src/GitVersionExe.Tests/VersionWriterTests.cs]: I could never het the linux travis ci to build. Did that ever get resolved?

Comment (JakeGinnivan) [inline src/GitVersionExe.Tests/VersionWriterTests.cs]: It didn't actually, I didn't notice that before merging. I likely will just not run this test on Mono. Thoughts @asbjornu

Comment (asbjornu) [inline src/GitVersionExe.Tests/VersionWriterTests.cs]: @JakeGinnivan: Running the build now to see what's up.

**Pull request merged**